package.json

```diff
@@ -2,16 +2,39 @@
   "name": "minecraft-data",
   "version": "2.70.0",
   "description": "Provide easy access to minecraft data in node.js",
-  "main": "index.js",
-  "tonicExampleFilename": "example.js",
+  "main": "build/main/index.js",
+  "typings": "build/main/index.d.ts",
+  "module": "build/module/index.js",
+  "tonicExampleFilename": "./src/example.js",
   "scripts": {
-    "generate:types": "node typings/generate-typings.js",
-    "generate:data": "./bin/generate_data.js",
-    "test:types": "tsc typings/test-typings && node typings/test-typings.js",
-    "test": "require-self && npm run generate:data && npm run lint && npm run test:types && mocha",
-    "lint": "standard",
-    "prepublish": "require-self && npm run generate:data && npm run generate:types",
-    "fix": "standard --fix"
+    "build": "run-p build:*",
+    "build:main": "tsc -p tsconfig.json",
+    "build:module": "tsc -p tsconfig.module.json",
```

Comment:
We definitely want to keep using standard not prettier
Look at the doc for how to use it

Reply:
It looks like standard does not support typescript so we could use eslint?

Reply:
See how it’s done here - https://github.com/PrismarineJS/mineflayer-statemachine/blob/standard-impl/package.json

Reply:
Yep I can move it to the ts-standard plugin